lib/guardian/post_guardian.rb

```diff
@@ -173,6 +172,11 @@ def can_edit_post?(post)
     false
   end
 
+  def can_edit_all_regular_posts?
+    SiteSetting.edit_all_post_groups.present? &&
```

Comment:
I might be wrong, but I don't think we need `SiteSetting.edit_all_post_groups.present?`